model-engine/model_engine_server/common/dtos/llms/model_endpoints.py

```diff
@@ -49,6 +49,16 @@ class CreateLLMModelEndpointV1Request(BaseModel):
     Path to the checkpoint to load the model from.
     """
 
+    chat_template: Optional[str] = Field(
```

Comment:
why do we have both chat_template and chat_template_override?

Reply:
oh woops, meant to delete this